tests/test_pruning.sh

```diff
@@ -4,13 +4,23 @@ set -xeuo pipefail
 # This test isn't standalone for now. It's executed from the `.cci.jenkinsfile`.
 # It expects to be running in cosa workdir where a single fresh build was made.
 
-# Test that first build has been pruned
-cosa build ostree --force-image # this is a trick to get a no-op new build
+# Test that first build has been pruned; create a few builds for testing.
+# FIXME: Add env COSA_BUILD_DUMMY=true or something since this test is very
+# expensive in CI; or better add `cosa build --shortcut=overrides` or so.
```

Comment:
Side note: this is actually not very expensive because `--force-image` only invalidates the image checksum state, but we're not actually rebuilding images at all since we do `cosa build ostree`. So this is close already to a no-op "dummy" build.

But yeah, agreed we could support this more properly.